cinema/tests/test_order_api.py

```diff
@@ -64,6 +64,7 @@ def test_get_order(self):
         self.assertEqual(ticket["row"], 2)
         self.assertEqual(ticket["seat"], 12)
         movie_session = ticket["movie_session"]
+        print(movie_session)
```

Comment:
Remove this print statement as it is not necessary for the test and can clutter the test output. Tests should generally avoid print statements unless they are specifically needed for debugging.